Account/FxALoginStateMachine.swift

```diff
@@ -115,6 +115,9 @@ class FxALoginStateMachine {
                                 log.error("Unknown error: \(error.description).  Transitioning to Separated.")
                                 return separated
                             }
+                        case let .Local(localError) where localError.domain == NSURLErrorDomain:
+                            log.warning("Local networking error: \(result.failureValue!).  Assuming transient and not transitioning.")
```

Comment:
Does this log the code? If not, let's do that too. (And eventually we want to record telemetry for this…!)